### PR TITLE
6 get

### PR DESCRIPTION
**Pull request by t-cbailey**
6. GET /api/reviews/:review_id/comments

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -84,4 +84,49 @@ describe("App", () => {
...
+    test("GET 200- array should be ordered by created_at ", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then((res) => {
+          const arr = res.body.comments;
+          expect(arr).toBeSortedBy("created_at");
+        });
+    });
+
```
Comment (HannahBurks) [inline __tests__/app.test.js:154]: I would like another test for an ID that is valid, but has no comments yet (so should return an empty array) - take a look at your data and see what ID has no comments :)

File: app_utils.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// const format = require("pg-format");
```
Comment (HannahBurks) [inline app_utils.js]: Make sure to remove any commented out code

File: output.txt [hunk 3]
```diff
@@ -1,6 +1,8 @@
 You are now connected to database "nc_games_test" as user "timbailey".
- review_id |  title   | category  |   designer    |    owner    |  review_body  |                                review_img_url                                |       created_at        | votes 
------------+----------+-----------+---------------+-------------+---------------+------------------------------------------------------------------------------+-------------------------+-------
-         1 | Agricola | euro game | Uwe Rosenberg | mallionaire | Farmyard fun! | https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700 | 2021-01-18 10:00:20.514 |     1
-(1 row)
+ comment_id | votes |       created_at       |     author      |                                                       body                                                       | review_id
```
Comment (HannahBurks) [inline output.txt]: Make sure to delete any unnecessary files from git hub and add to git ignore

File: models.js [hunk 4]
```diff
@@ -17,3 +17,19 @@ exports.findReviewByID = (id) => {
...
+exports.findCommentsByRevID = (id) => {
+  return connection
+    .query(
+      `SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.review_id FROM reviews 
+  JOIN comments ON comments.review_id = reviews.review_id
+  WHERE reviews.review_id = $1
+  ORDER BY comments.created_at;`,
+      [parseInt(id.review_id)]
+    )
+    .then((res) => {
```
Comment (HannahBurks) [inline models.js]: Think about how we can check whether the reason for no data back is due to the article ID not existing or whether it is due to there just not being any comments yet. What do we already have that exists, that would allow us to check this?

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Nice work so far Tim, just a few things to add/think about in regards to tests and checking reason for returning no data (due to non existent ID or no comments)

**Review by HannahBurks — APPROVED** [review]
Excellent, feel free to merge. Can you please also ensure to update your endpoint JSON and put the output.txt and playground.txt in your git ignore and delete from git hub 😄

**Comment by t-cbailey** [comment]
Thanks Hannah! 

I have put the playground and output into git ignore for every push- but hadn’t deleted from GH. Maybe that’s why it’s still there!

Tim
